templates/partial/category-index.html.twig

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
This file name seems off to me. Should it be term-index.html.twig? it deals with terms, which might not be categories. I guess they're all different ways of categorizing things, but we could eventually have something called "category" and then this gets difficult. I'm not sure it's worth changing right now.

Reply:
Good call--I think in terms of document categories, but you're right that that doesn't describe what these things are properly. I've changed that now.